.github/actions/devcontainer_run_command/action.yml

```diff
@@ -45,10 +45,10 @@ inputs:
   ARM_SUBSCRIPTION_ID:
     description: "Azure Subscription Id."
     required: true
-  RESOURCE:
+  API_CLIENT_ID:
     description: "The API Client Id."
     required: true
-  AUTH_TENANT_ID:
+  AAD_TENANT_ID:
     description: "The Tenant Id where the App is registered."
```

Comment:
Isn't this the tenant of the TEST_USER?
If yes, maybe a better name and description is in order?

Reply:
Yes, but they are the same thing. It's confusing because we were copying AAD_TENANT_ID into AUTH_TENANT_ID. This means that we can set one value in /workspaces/AzureTRE/templates/core/.env as per the docs and the tests just work